src/payment/payment-action-creator.ts

```diff
@@ -39,17 +41,22 @@ export default class PaymentActionCreator {
         methodId: string,
         gatewayId?: string,
         instrumentId?: string,
-        shouldSaveInstrument?: boolean
+        shouldSaveInstrument?: boolean,
+        browserInfo?: BrowserInfo
     ): ThunkAction<InitializeOffsitePaymentAction, InternalCheckoutSelectors> {
         return store => {
-            let paymentData: FormattedPayload<FormattedHostedInstrument | FormattedVaultedInstrument> | undefined;
+            let paymentData: FormattedPayload<FormattedBrowserInfo | FormattedHostedInstrument | FormattedVaultedInstrument> | undefined;
```

Comment:
If I read this correctly, we are saying that you could never send `FormattedHostedInstrument` or `FormattedVaultedInstrument` together with `FormattedBrowserInfo` but your code seems to be merging them together.